server/src/services/blogs/render.js

```diff
@@ -1,26 +1,30 @@
 import env from '../../configs/env';
 
 const shareBlog = async (req, res) => {
-    const {
-        _id,
-        title= '',
-     content = '',
-      imageUrl = 'https://cdn3.wpbeginner.com/wp-content/uploads/2016/11/blogimagetools.jpg',
-      isPrivate = true,
-      published = false,
-     } = req.blog;
-     const isSharable = !isPrivate && published
-     if(true) {
-         const redirectUrl = `${env.appViewUrl}blog/${_id}`
-        res.render('shareBlog', {
-            title,
-            content, 
-            imageUrl,
-            redirectUrl,
-        });
-     }
-     res.status(400).send('Not Found');
-  };
-  
+  const {
+    _id,
+    title = '',
+    description = '',
+    imageUrl = 'https://cdn3.wpbeginner.com/wp-content/uploads/2016/11/blogimagetools.jpg'
+  } = req.blog;
+  const prependPath = `${env.appViewUrl}blog`;
+  const draftPath = `${prependPath}/draft/${req.params.draftUrl}/${_id}`;
+  const privatePath = `${prependPath}/secured/${req.params.draftUrl}/${_id}`;
```

Comment:
Fix this error.

Reply:
Can you elaborate what error are you talking about ? :smile: